js/augurs-prophet-js/src/lib.rs

```diff
@@ -1010,6 +1036,42 @@ impl From<(Option<f64>, augurs_prophet::Predictions)> for Predictions {
     }
 }
 
+/// Options to specify whether to include the historical dates in the future dataframe for predictions.
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Deserialize, Tsify)]
+#[serde(rename_all = "camelCase")]
+#[tsify(from_wasm_abi, type_prefix = "Prophet")]
+pub struct IncludeHistoryOptions {
```

Comment:
I think this type should be `MakeFutureDataframeOptions`, where currently the only field is `include_history` but we have flexibility to add more in future. Otherwise it looks good.